project/legislation/legislatorslocation.py

```diff
@@ -0,0 +1,35 @@
+import Model from model
```

Comment:
For file names with multiple words, separate the words with underscores. This makes it easier to read the file name.